plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -92,18 +100,17 @@ private DeltaLakeSchemaSupport() {}
     public static final String MAX_COLUMN_ID_CONFIGURATION_KEY = "delta.columnMapping.maxColumnId";
     private static final String DELETION_VECTORS_CONFIGURATION_KEY = "delta.enableDeletionVectors";
 
-    // https://github.com/delta-io/delta/blob/master/PROTOCOL.md#valid-feature-names-in-table-features
     private static final Set<String> SUPPORTED_READER_FEATURES = ImmutableSet.<String>builder()
-            .add("columnMapping")
-            .add("timestampNtz")
-            .add("deletionVectors")
+            .add(COLUMN_MAPPING.getFeatureName())
```

Comment:
if the code doesn't benefit from enum type, it would be easier to define a bunch of `public static final String` constants.

i think we can do better -- should SUPPORTED_READER_FEATURES be declared as `Set<TableFeature>`?

Reply:
Changed to String constants. Even if we change `SUPPORTED_READER_FEATURES ` to `Set<TableFeature>`, we need to convert it to String in `unsupportedReaderFeatures` method because the table may contains undeclared table features.